src/IBM.Watson.SpeechToText.v1/IBM.Watson.SpeechToText.v1.csproj

```diff
@@ -35,8 +35,9 @@
 
     <ItemGroup>
         <PackageReference Include="IBM.Cloud.SDK.Core" Version="1.2.0" />
-        <PackageReference Include="IBM.Watson.Common" Version="5.1.0" />
+        <PackageReference Include="IBM.Watson.Common" Version="5.2.0" />
         <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
-        <PackageReference Include="JsonSubTypes" Version="1.8.0" />

```

Comment:
Why was this version reduced?

Reply:
hmm odd, im not sure why it is using 1.8.0, ill leave it as 1.8.0 then